### PR TITLE
Create issue with default fields

### PR DESCRIPTION
**Pull request by zpawn**
Story https://app.shortcut.com/deskpro/story/98883/youtrack-create-an-issue

### CODE REVIEW

File: src/services/youtrack/constants.ts [hunk 1]
```diff
@@ -18,7 +18,42 @@ export const COMMENT_FIELDS = [
...
 ];
 
-export const PROJECT_FIELDS = ["id", "shortName", "name"];
+export const PROJECT_CUSTOM_FIELDS = [
+  "id",
+  `field(${[
+    "id",
+    "name",
+    "localizedName",
+    "fieldType(valueType,id)", // ./src/components/ViewIssue/IssueFieldView/types.ts:MappingFieldTypes
```
Comment (amDavid01) [inline src/services/youtrack/constants.ts:27]: uhh whats this

File: src/pages/CreateIssuePage/__tests__/CreateIssuePage.test.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import { cleanup, act } from "@testing-library/react";
```
Comment (amDavid01) [inline src/pages/CreateIssuePage/__tests__/CreateIssuePage.test.tsx:1]: Are you testing the functionality of this page?

File: src/components/IssueItem/__tests__/IssueItem.test.tsx [hunk 3]
```diff
@@ -3,9 +3,8 @@ import { cleanup } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
```
Comment (amDavid01) [inline src/components/IssueItem/__tests__/IssueItem.test.tsx:3]: Are you testing the functionality of this page? you should be checking for example if SPDP-3 or Issue is inside the page

File: src/components/IssueForm/__tests__/IssueForm.test.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import { cleanup, act } from "@testing-library/react";
```
Comment (amDavid01) [inline src/components/IssueForm/__tests__/IssueForm.test.tsx:1]: Are you testing the functionality of this page? Like what if everything is ok? You could make a check to see if the submit function is being called with all the required data

File: src/components/IssueItem/__tests__/IssueItem.test.tsx (hunk 3 above)
Comment (zpawn) [reply src/components/IssueItem/__tests__/IssueItem.test.tsx:3]: Yes. I tested the IssueForm component by passing mocked `onSubmit` function to it

File: src/pages/CreateIssuePage/__tests__/CreateIssuePage.test.tsx (hunk 2 above)
Comment (zpawn) [reply src/pages/CreateIssuePage/__tests__/CreateIssuePage.test.tsx:1]: Yes. I tested the IssueForm component by passing mocked onSubmit function to it

File: src/components/IssueItem/__tests__/IssueItem.test.tsx (hunk 3 above)
Comment (zpawn) [reply src/components/IssueItem/__tests__/IssueItem.test.tsx:3]: This is checked in this file

File: src/components/IssueForm/__tests__/IssueForm.test.tsx (hunk 4 above)
Comment (zpawn) [reply src/components/IssueForm/__tests__/IssueForm.test.tsx:1]: Yes. It's tested. The form consists of many components and utilities. They are all tested. And for the form itself, I wrote a couple of basic tests

**Review by amDavid01 — APPROVED** [review]

File: src/services/youtrack/constants.ts (hunk 1 above)
Comment (zpawn) [reply src/services/youtrack/constants.ts:27]: I'll remove it in the next PR)))

**Review by AshleyDawson — APPROVED** [review]

**Pull request merged**